### ISSUE TITLE
Feature Request for delete-task and listing the tasks #83770848 #84348820

### ISSUE THREAD

**Issue by asifhuawei** [cloudfoundry-incubator/lattice#124]
Hi @davidwadden as per discussion in one of the previous [issues] (https://github.com/cloudfoundry-incubator/lattice/issues/117#issuecomment-104915581) I have implemented the new command to delete the task ```ltc delete-task``` and to list the tasks in ```ltc list```.
Here is the quick view of my implementation
```
root@ubuntu:~# ltc delete-task -h
NAME:
   delete-task, dt - Deletes the given task

USAGE:
   ltc delete-task TASK_NAME

root@ubuntu:~# ltc list
==========================================================================================
      Displaying Apps
------------------------------------------------------------------------------------------
App Name        Instances       DiskMB          MemoryMB        Route
lattice-app     0/1             0               128             lattice-app.192.168.11.11.xip.io, lattice-app-8080.192.168.11.11.xip.io => 8080
------------------------------------------------------------------------------------------


==========================================================================================
      Displaying Tasks
------------------------------------------------------------------------------------------
Task Name       Cell ID         Status          Result          Failiure Reason
ltc-EchoTask    lattice-cell-01 COMPLETED       -NA-            failed to initialize container
ltc-SleepTask   lattice-cell-01 COMPLETED       -NA-            failed to initialize container
------------------------------------------------------------------------------------------

root@ubuntu:~# ltc delete-task ltc-EchoTask
Deleting the task ltc-EchoTask
OK
root@ubuntu:~#
root@ubuntu:~#
root@ubuntu:~# ltc delete-task InvalidTaskGuid
Deleting the task InvalidTaskGuid
Error Deleting the task InvalidTaskGuid
Failiure Reason :Task not found.

root@ubuntu:~#

``` 
![image](https://cloud.githubusercontent.com/assets/9736066/7798298/ef65d116-0315-11e5-8eeb-16707d772c38.png)

Please give us some feedback, we will be happy to make changes. If it feels ok then we can raise a PR for this.

Thanks & Regard
Asif

**Comment by davidwadden** [comment]
A couple little things:

1. Change `-NA-` to `N/A` 
2. Spelling on "Failiure"
3. On the titles, replace "Displaying Apps/Tasks" with just "Apps/Tasks"

Also, there's a lot of space lost to the separators.  Maybe try and see if it could be done with just
`-----------= Tasks =--------------------` ?

Go ahead and put the PR in so we can take a look at the code too

**Comment by asifhuawei** [comment]
Thanks @davidwadden for a quick feedback, will try to make changes and raise a PR by the end of the day.

**Comment by menicosia** [comment]
We've released Version 0.2.5 of Lattice, which includes Tasks! I've been keeping this issue open until that release.

https://github.com/cloudfoundry-incubator/lattice/releases/tag/v0.2.5

Thanks for the Pull Requests!

Closing this issue, but please feel free to reopen if this release does not sufficiently address this issue.

--
  Marco Nicosia
  Product Manager
  Pivotal Software, Inc.
